compiler/rustc_mir_build/src/check_unsafety.rs

```diff
@@ -148,11 +152,28 @@ impl<'a, 'tcx> Visitor<'a, 'tcx> for UnsafetyVisitor<'a, 'tcx> {
             ExprKind::Call { fun, ty: _, args: _, from_hir_call: _, fn_span: _ } => {
                 if self.thir[fun].ty.fn_sig(self.tcx).unsafety() == hir::Unsafety::Unsafe {
                     self.requires_unsafe(expr.span, CallToUnsafeFunction);
+                } else if let &ty::FnDef(func_did, _) = self.thir[fun].ty.kind() {
```

Comment:
as far as I know, target features 1.1 is not stable https://github.com/rust-lang/rust/issues/69098 -- is this instability enforced at the declaration site? I guess probably so.

Reply:
Yes, the check is here: https://github.com/rust-lang/rust/blob/1773f14a24c49356b384e45ebb45643bc9bef2c4/compiler/rustc_typeck/src/collect.rs#L2773-L2781